dev-python/regex/regex-2019.20.12.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="All things regular expression: implemented in python"
+DESCRIPTION="Regular expression module, to replace re"
+HOMEPAGE="https://pypi.org/project/regex/"
+COPYRIGHT="2013-2019 Matthew Barnett"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://pypi.org/packages/source/r/regex/regex-$portVersion.tar.gz"
+CHECKSUM_SHA256="106e25a841921d8259dcef2a42786caae35bc750fb996f830065b3dfaa67b77e"
+SOURCE_DIR="regex-2019.12.20"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
```

Comment:
Missing something here to be able to build it for secondary architectures

Reply:
like now?